src/tests/ftest/telemetry/pool_space_metrics.py

```diff
@@ -119,13 +118,12 @@ def test_telemetry_pool_space_metrics(self):
             self.add_container(pool=self.pool)
 
             # Run ior command.
-            try:
-                self.update_ior_cmd_with_pool(create_cont=False)
-                self.run_ior_with_pool(
-                    timeout=200, create_pool=False, create_cont=False)
-            except TestFail as error:
-                self.log.info("ior command failed: %s", str(error))
-                raise
+            self.update_ior_cmd_with_pool(create_cont=False)
+            self.run_ior_with_pool(
+                timeout=200, create_pool=False, create_cont=False)
+
+            self.log.info("Waiting 15s let the vos space metrics to be updated")
+            time.sleep(15)
```

Comment:
If it's straightforward to update the code, it would be better to use a while loop with 15 sec delay for a few iterations. Then place line 128 to 141 inside the loop. If line 135 passes for all `self.metric_names`, break the loop and we're done. Otherwise, fail the test.

Reply:
Make sense: CI run time are always good to take for everyone ;)
I will update the code in this way. I will also take the opportunity to make it a little bit more reliable: breaking the loop if the test are successfully completed two consecutive times.

- [ ] Remove 15s time sleep with monitoring loop